docs/api/1.x/accounts.rst

```diff
@@ -28,6 +28,9 @@ Add the following settings to the ``.ini`` file:
     # Allow anyone to create accounts.
     kinto.account_create_principals = system.Everyone
 
+    # Set the session time to live in seconds for the session
```

Comment:
nit: `session time` and `for the session` does not sound super clear in the same sentence :)